grisera_api/graph_api_service.py

```diff
@@ -56,7 +56,7 @@ def create_properties(self, node_id: int, node_model: BaseModel):
         for key, value in node_dict.items():
             if key == 'additional_properties' and value is not None:
                 request_body.extend(self.create_additional_properties(property_dict=node_dict))
-            elif not isinstance(value, list) and not isinstance(value, dict):
+            elif not isinstance(value, list) and not isinstance(value, dict) and value is not None:
```

Comment:
I think it would make more sense if we reverse the order -- if `value` is null, we don't have to check if it's not a `list` or `dict`.